packages/cli/src/utils/bundle.ts

```diff
@@ -166,13 +173,17 @@ export async function bundle(dirPath: string, options?: { outfile?: string }) {
         'wasi',
         'worker_threads',
         'zlib',
+        'chromium-bidi/lib/cjs/bidiMapper/BidiMapper',
```

Comment:
do we still need these additions

Reply:
yeah we still need this
this is accounted for by `@mastra/rag`